src/server/worldserver/worldserver.conf.dist

```diff
@@ -2516,6 +2519,14 @@ Rate.MoveSpeed = 1
 
 Rate.InstanceResetTime = 1
 
+#
+#    Rate.Pet.LevelXP
```

Comment:
hmm why not put in Rate.XP.Pet.PlayerPercentage instead (so all in same code block)
Then you'd rename Rate.XP.Pet to Rate.XP.Pet.Raw

Maybe i'm wrong, just suggesting